src/marqo/tensor_search/enums.py

```diff
@@ -119,9 +119,15 @@ class RequestType:
     DELETE = "DELETE"
     CREATE = "CREATE"
 
+# Each type has different add_documents behavior. More can be added to represent special types.
+class DocumentFieldType:
```

Comment:
Do we have tests about whether these are translate-able into strings, if that is desired behaviour?